internal/integration/helpers.go

```diff
@@ -102,7 +125,20 @@ func testServer(tb testing.TB) (*serverenv.ServerEnv, *http.Client) {
 		tb.Fatal(err)
 	}
 
-	bs, err := storage.NewMemory(ctx)
+	var bs storage.Blobstore
+	var b [512]byte
```

Comment:
Can you make the random generation a helper please?

Reply:
done